app/src/main/java/snp/zararoid/ui/main/product/ProductListFragment.kt

```diff
@@ -1,19 +1,34 @@
 package snp.zararoid.ui.main.product
 
+import android.content.Context
 import android.os.Bundle
-import androidx.fragment.app.Fragment
+import android.util.Log
 import android.view.LayoutInflater
 import android.view.View
 import android.view.ViewGroup
+import android.widget.Toast
+import androidx.fragment.app.Fragment
 import androidx.recyclerview.widget.GridLayoutManager
 import kotlinx.android.synthetic.main.fragment_product_list.*
+import okhttp3.ResponseBody
+import org.json.JSONObject
+import retrofit2.Call
+import retrofit2.Callback
+import retrofit2.Response
 import snp.zararoid.R
 import snp.zararoid.ui.main.product.rcv.ProductAdapter
 import snp.zararoid.ui.main.product.rcv.ProductData
+import snp.zararoid.ui.network.ProductListResponseData
+import snp.zararoid.ui.network.ZaraReDesign
+
 
 class ProductListFragment : Fragment() {
     private lateinit var productAdapter: ProductAdapter
     private lateinit var rcvLayoutManager: GridLayoutManager
+    private var category: String = ""
+    private var rcvProductListData = mutableListOf<ProductData>()
```

Comment:
```suggestion
    private val rcvProductListData = mutableListOf<ProductData>()
```
mutableList는 개인적으로 var일 필요가 없다고 생각해 자체적으로 추가,수정,삭제가 가능하다면 새로운놈으로 대체하는것보단 기존 값을 처리해주는게 좋지 않을까..? 하는 생각? 일단 지금 코드에선 해당 리스트가 변하지 않는거 같아서 val을 추천하는 부분이야!